test/base/exception/KintoneAPIException.test.js

```diff
@@ -0,0 +1,89 @@
+import KintoneAPIException from '../../../src/base/exception/KintoneAPIException';
+import ErrorResponse from '../../../src/base/exception/ErrorResponse';
+
+describe('Checking KintoneAPIException object', () => {
+  it('Can create new instance of KintoneAPIException with Buffer data', () => {
+    const errors = {
+      response: {
+        data: Buffer.from('hello buffer')
+      }
+    };
+
+    const kException = new KintoneAPIException(undefined, errors);
+    expect(kException).toBeInstanceOf(KintoneAPIException);
+  });
+
+  it('Can create new instance of KintoneAPIException with ArrayBuffer data', () => {
+    const errors = {
+      response: {
+        data: new ArrayBuffer('hello buffer')
+      }
+    };
+
+    const kException = new KintoneAPIException(undefined, errors);
+    expect(kException).toBeInstanceOf(KintoneAPIException);
+  });
+
+  it('Can getOriginError()', () => {
+    const errors = new Error('ERROR_MESSAGE');
+    const kException = new KintoneAPIException(undefined, errors);
+
+    const originError = kException.getOriginError();
+    expect(originError.message).toEqual('ERROR_MESSAGE');
+  });
+
+  it('Can getErrorResponse()', () => {
+    const errors = new Error('ERROR_MESSAGE');
+    const kException = new KintoneAPIException(undefined, errors);
+
+    const errorResp = kException.getErrorResponse();
+    expect(errorResp.message).toEqual('ERROR_MESSAGE');
+  });
+
+  it('Can getHttpErrorCode()', () => {
+    const errors = {
```

Comment:
refactor this "errors" to "httpResponse"
and also change "statusCode: 200" to "status: 200"